IBPSA/ThermalZones/ISO13790/Data/Generic.mo

```diff
@@ -0,0 +1,22 @@
+within IBPSA.ThermalZones.ISO13790.Data;
+record Generic "Generic data record building mass"
+   extends Modelica.Icons.Record;
+
+   parameter Real heaC
```

Comment:
This is not clear, in part because the unit is missing and the documentation does not explain what capacity it is. It is used in 
```modelica
  Modelica.Thermal.HeatTransfer.Components.HeatCapacitor capMas(
      C=buiMas.heaC*AFlo,
      T(displayUnit="degC",
      fixed=true,
      start=293.15)) "Zone thermal capacity"
``` and hence it looks like a capacity per unit area of the floor.

Reply:
I agree with all changes. The description of `Data.Generic` has been improved. It is a heat capacity per unit area of the floor (J/K m2)

Reply:
Thanks, looks good, but I suggest you change "Heat mass" to "Thermal mass" as the latter is customary.

Reply:
Not completely sure about what you mean, but I changed the description of `Data.Generic` from "Generic data record building mass" to "Generic data record for building thermal mass". Please let me know if this is what you meant.

Reply:
I updated the data record description in my latest commit to the pull request from issue1492_reducedOrderISO13790_rev_2_mw